modules/telco-ran-workload-partitioning.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+
+:_content-type: REFERENCE
+[id="telco-ran-workload-partitioning_{context}"]
+= Workload partitioning
+
+Description::
+Workload partitioning pins {product-title} platform and Day 2 Operator Pods that are part of the DU profile to the reserved `cpuset` and removes the reserved CPU from node accounting.
+This leaves all unreserved CPU cores available for user workloads.
++
+The method of enabling and configuring Workload partitioning changed in {product-title} 4.13.
++
+--
+4.12 and earlier::
+* Partitions are set with extra `MachineConfiguration` CRs applied at install-time
+
+4.13 and later::
+
+* Partitions are enabled by setting the capability in the cluster installation parameters
+
+* Management partition cores are set with the reserved CPU in the `PerformanceProfile` CR
+--
+
+Limits and requirements::
+* `Namespace` and `Pod` CRs must be annotated to allow the Pod to be applied to the management partition
+
+* Pods with CPU limits cannot be allocated to the partition.
+This is because mutation can change the Pod QoS.
+
+* For more information about the minimum number of CPUs that can be allocated to the management partition, see xref:../../telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc#telco-ran-node-tuning-operator_ran-ref-design-components[Node Tuning Operator].
```

Comment:
No xrefs in modules. Move to the assembly as addtnl resources section.
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#internal-cross-references